### DISCUSSION TITLE
[Idea]: Predefined Video Playback Before Meetings and Assemblies

### DISCUSSION THREAD

**Discussion by bucmado** [sircharlo/meeting-media-manager#3851] [Ideas] [answered]
### Is your idea related to a problem?

N/A

### Describe your idea

It would be excellent to include an optional feature in the M3 system that allows the automatic playback of a predefined video before meetings or assemblies begin. The idea is for the system to calculate the duration of the video and play it automatically so that it finishes right at the start of the meeting.  

### Feature Details:  
1. **Predefined Video**:  
   - Allow the upload of a fixed-duration video, such as 10 minutes (for assemblies) or 1 minute (for regular meetings).  

2. **Automatic Start**:  
   - The system would automatically play the video before the meeting, calculating the required start time to ensure it ends precisely as the activity begins.  
   - For example, if the video is 10 minutes long, the system would begin playback exactly 10 minutes before the scheduled start time.  

3. **Background Music Integration**:  
   - For assemblies, the system could keep background music playing until 1 minute before the start time. Then, the music would automatically stop, and the video would finish playing just as the meeting begins.  

### Benefits:  
- **Avoid Manual Oversights**: Currently, we play a countdown video manually, and while this works well, the person handling audio/video may sometimes forget to start it.  
- **Consistency in Assemblies and Meetings**: This would be particularly useful during assemblies and meetings with larger audiences, ensuring everything flows automatically.  
- **System Expansion**: We have already demonstrated the M3 system to brothers working at LDCs, who have expressed interest in implementing it in more congregations and assemblies. Including this functionality would be a valuable improvement for broader adoption.  

We greatly appreciate your consideration of this suggestion, as it would further simplify system use and enhance its utility for congregations and assemblies.  

https://github.com/user-attachments/assets/1b43cd71-2768-4528-a1a7-d0bb7777279e

### Describe alternatives you've considered

N/A

### Additional context

N/A

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my idea is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my idea is not a duplicate.

**Comment by sircharlo** [comment]
Are these introductory videos provided by the branch? I hesitate to implement features such as the ones you describe because our current direction (in Canada, in any case) stipulates that no moving footage (ie: videos) should be displayed unless provided by the branch, and no such introductory videos are provided here for meetings or assemblies.

**Comment by bucmado** [comment]
Thank you for your response.

Here in the Dominican Republic, a countdown timer is used during assemblies along with the prelude music. In some cases, a video with images or motion footage accompanied by music is also shown. However, when such a video is not prepared, the countdown timer is displayed along with the music to encourage the brothers to take their seats before the program begins.

In fact, last week, the brothers in charge of audio and video for the assembly contacted me to request the countdown timer for this purpose. In assemblies, a 10-minute countdown timer is typically used before the start of the program. It is worth noting that this timer is only displayed at the beginning of the assembly. Once the program begins, only the year text remains on the screen, with no additional changes unless scheduled content is projected for publications or public talks.

Similarly, in congregations, if the elders deem it appropriate or necessary, the countdown timer is used. It is not implemented in all congregations but is utilized in those where the body of elders considers it beneficial. In my congregation, for example, we only use a 1-minute countdown timer before the meeting begins.

I hope this clarifies how this feature is used locally and why it could be beneficial to include it as an option in the M3 system.

**Reply by sircharlo** [reply, marked as answer]
### For assemblies

> Here in the Dominican Republic, a countdown timer is used during assemblies along with the prelude music. 

In our branch territory, the CO-160 is very clear in stating that only media files prepared by the branch for the assembly are to be displayed, specifically including the time from the opening of the doors until the start of the morning session. (CO-160 4:8) The only exception made in the direction is for video during the baptism and for interpreted talks, as well as for the use of images during the Watchtower summary. (CO-160 4:12-13)

Does your direction differ? Is this countdown timer prepared by your local branch? 

### For congregation meetings

> Similarly, in congregations, if the elders deem it appropriate or necessary, the countdown timer is used.

The direction found in _sfl_ 20:23, combined with that found in S-141 8-9, is what informed the decision to not implement the countdown timer feature for meetings in M³. This direction indicates that displaying videos that were created locally or that were not provided by the branch is not allowed.

Does your branch territory have different direction or specific instructions that supercede this direction?

**Reply by bucmado** [reply]
Thank you for the information.

I took the time to carefully read all the referenced instructions you mentioned, together with a congregation elder who also works at the LDC. We came to the same conclusion that you highlighted.

Based on this, we will proceed to stop using the countdown timer in our congregations and ensure we keep the references on hand for any future requests regarding this content or if any brother has questions. This will also be shared with those in charge of audio and video at the assembly.

Thank you again for pointing us in the right direction.

**Reply by sircharlo** [reply]
It's a pleasure to work alongside you, dear brothers!
